webrob/startup/init_app.py

```diff
@@ -21,6 +21,8 @@
 
 from werkzeug.security import generate_password_hash, check_password_hash
 
+admin_user_default_pw = '1234'
```

Comment:
I think, usually you would use uppercase-only for constants, but this is not so important